pypeclub/OpenPype
# Support for Unreal 5

## Fixing support for Unreal 5

This is adding support for both UE4 and UE5, dealing with the need for two versions of integration plugin and adding `UE_PYTHONPATH` to defaults for UE5.

There is still one caveat and that is that we need to install pyside2 to Unreal and we can't do that when Unreal is installed to privileged location (like Program Files on Windows) where you need admin rights to write.

### To test it:
Install both UE4 and UE5 versions, enable them on project, make sure you have `{"UE_PYTHONPATH": "{PYTHONPATH}"}` in environment for UE5 and try to create projects in both version.

✏️ Note: You'll need Visual Studio if you don't have already built integration plugins.

Close #3101
